### PR TITLE
Add Smoove private domain

### PR DESCRIPTION
**Pull request by DanSmoove**
* [x] Description of Organization
* [x] Reason for PSL Inclusion
* [x] DNS verification via dig
* [x] Run Syntax Checker (make test)

* [x] Each domain listed in the PRIVATE section has and shall maintain at least two years remaining on registration, and we shall keep the _PSL txt record in place

__Smoove company affirms the following:__ 
  * [x] We are listing any third party limits that we seek to work around in our rationale such as those between IOS 14.5+ and Facebook (see [Issue #1245](https://github.com/publicsuffix/list/issues/1245) as a well-documented example)
  * [x] This request was _not_ submitted with the objective of working around other third party limits
  * [x] The [Guidelines](https://github.com/publicsuffix/list/wiki/Guidelines) were carefully _read_ and _understood_, and this request conforms
  * [x] The submission follows the [guidelines](https://github.com/publicsuffix/list/wiki/Format) on formatting
 * [x] Yes, I understand.  I could break my organization's website cookies etc. and the rollback timing, etc is acceptable.  Proceed.
---

Description of Organization
====

Organization Website: https://www.smoove.io 
We are an ESP and marketing platform that gives businesses from all over the world a full set of tools for their marketing and administrative needs.
from newsletters to landing pages to full marketing automation.

Reason for PSL Inclusion
====

Our users get a vp4.me subdomain for their landing pages, the subdomain is personal and separate for each account.
they use it for their business needs: various registration forms, purchase pages, advertising campaigns, business cards, and more.
for example: danafb1.vp4.me

This is a pull request in response to the Apple iOS 14.5 change that affected Facebook Pixel tracking functionality for the domain(s) in the change, and we seek to resolve the impacts to the best of our ability to do so. 
Both Apple and Facebook support Public Suffix List for Private Click Measurement and Aggregated Event Measurement respectively.
We have reviewed the wiki here outlining the PSL and understand what it is and is not.
We have reviewed this matter with the team at Facebook, we are now fully aware about the consequences to the change we are requesting as it relates to cookies or other services, and choose to proceed in submitting this PR. We understand that despite reviewing the matter with Facebook, Facebook cannot control which use cases get accepted to the PSL We understand that there is not any urgency and should this PR be merged to include our requested change, that there is no control over the timing of this change being updated downstream in browsers, libraries, certificate authorities or other systems or processes that incorporate the PSL.
We understand that if we later want to back out this change, there is also no guarantee that this will occur in a rapid fashion due to the same lack of control over browser, library, certificate authority, or other system or process changes.

DNS Verification via dig
=======
dig +short TXT _psl.vp4.me
"https://github.com/publicsuffix/list/pull/1351"

make test
=========

ran the test (syntax cheker):
test succeded
test output:
test_allowedchars: OK
test_dots: OK
test_duplicate: OK
test_exception: OK
test_NFKC: OK
test_punycode: OK
test_section1: OK
test_section2: OK
test_section3: OK
test_section4: OK
test_spaces: OK
test_wildcard: OK

### CODE REVIEW

**Comment by dnsguru** [comment]
#1328 (related)

**Comment by bedfordsean** [comment]
I can confirm that from the Facebook side we have been through the review process, informed Smoove of the potential issues and concerns with adding domains to the PSL, and also confirmed that adding vp4.me has significant existing volume of subdomains each of which appear to represent separate business entities.  Our recommendation to Smoove was to raise a request here as the outcome of that review.

**Comment by DanSmoove** [comment]
Hello @bedfordsean @dnsguru,
I wanted to know if there are news regarding the PR status.
Thanks, guys

**Comment by DanSmoove** [comment]
Hello @bedfordsean @dnsguru,
any news/updates?

**Comment by bedfordsean** [comment]
Nothing more for the Facebook side to do here - @dnsguru can confirm that Smoove have followed our intake form process and we've done the "legitimate use case" pre-checks already which led to the recommendation of raising a request here.

**Comment by dnsguru** [comment]
was #1328

File: public_suffix_list.dat [hunk 1]
```diff
@@ -13690,4 +13690,8 @@ basicserver.io
 virtualserver.io
 enterprisecloud.nu
 
+// Smoove.io : https://www.smoove.io/
```
Comment (dnsguru) [inline public_suffix_list.dat:13693]: Move this up higher in the PRIVATE DOMAINS section so that it is alphabetically ordered by your org's name please

Comment (DanSmoove) [reply public_suffix_list.dat]: Done 👍

File: public_suffix_list.dat [hunk 2]
```diff
@@ -6058,6 +6058,10 @@ org.sl
 // sm : https://en.wikipedia.org/wiki/.sm
 sm
 
+// Smoove.io : https://www.smoove.io/
```
Comment (dnsguru) [inline public_suffix_list.dat:6061]: Wrong section; Within the PRIVATE DOMAINS section not the ICANN ones please

**Comment by dnsguru** [comment]
Approving this request as a pilot / example while the #1245 solutions are evolving with resourcing help

**Pull request merged**